composer.json

```diff
@@ -3,23 +3,27 @@
   "description": "Official PHP client to the Killbill HTTP API",
   "type": "library",
   "autoload": {
-      "psr-4": {
-          "Killbill\\Client\\": "src/"
-      }
+    "psr-4": {
+      "Killbill\\Client\\": "src/"
+    }
   },
   "autoload-dev": {
-      "psr-4": {
-          "Killbill\\Client\\": "test/"
-      }
+    "psr-4": {
+      "Killbill\\Client\\": "test/"
+    }
   },
   "scripts": {
-      "test": "ENV=local phpunit",
-      "test-integration": "phpunit",
-      "record-mocks": "composer run test-integration && RECORD_REQUESTS=1 phpunit"
+    "test": "ENV=local phpunit",
+    "test-integration": "phpunit",
+    "record-mocks": "composer run test-integration && RECORD_REQUESTS=1 phpunit",
+    "phpcs": "./vendor/squizlabs/php_codesniffer/scripts/phpcs --standard=vendor/escapestudios/symfony2-coding-standard/Symfony2 ./src/ ./test/ --extensions=php --ignore=src/Type -d memory_limit=1024M",
```

Comment:
You can use here just `phpcs` instead of the whole path. `vendor/bin/` is being prepended to the `$PATH` by Composer.